docs/running-on-yarn.md

```diff
@@ -430,6 +430,21 @@ To use a custom metrics.properties for the application master and executors, upd
   See <code>spark.yarn.config.gatewayPath</code>.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.custom.log.url</code></td>
+  <td>(none)</td>
+  <td>
+  Specifies custom spark log url for supporting external log service rather than NodeManager webapp address.
+  Spark will support some path variables via patterns. Supported patterns and allocated values are below: <p/>
+  <p/>
+  * `{{HttpScheme}}`: `http`/`https` according to YARN HTTP policy. (Configured via `yarn.http.policy`)<p/>
+  * `{{NodeHttpAddress}}`: HTTP URI of the node on Container.<p/> 
```

Comment:
If this the full address of the NM HTTP server? Because it feels a little conflicting with the above variable.

Also I'm not sure what "node on container" means. Perhaps "node where container was run".